bundles/org.eclipse.equinox.console.ssh.tests/META-INF/MANIFEST.MF

```diff
@@ -13,6 +13,7 @@ Import-Package: org.apache.sshd.client,
  org.junit;version="4.8.1",
  org.mockito,
  org.mockito.stubbing,
- org.mockito.invocation
+ org.mockito.invocation,
+ org.osgi.service.cm
 Fragment-Host: org.eclipse.equinox.console.ssh, org.eclipse.equinox.console
```

Comment:
Does it really work to specify multiple host bundles for a fragment?
I've never seen this before but when thinking about it, at least the Wiring API allows an arbitrary number of host wires. But until now I just thought it was an hypothetical scenario due to the generality of the Wiring API.

Reply:
Last time I touched that part was for another build breakage probably. I wish I could help more but ... :)